src/bp/ui-studio/src/web/components/Confusion/.gitignore

```diff
@@ -0,0 +1 @@
+style.scss.d.ts
```

Comment:
Please commit those files, otherwise you will have TS errors in the module until it's built and the files are generated again